src/components/Scoreboard.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+
+function Scoreboard({winner}) {
+  const [scores, setScores] = useState({
+    X: 0,
+    O: 0,
+  });
+
+  useEffect(() => {
+    if (winner === 'X') {
+      setScores(prevScores => ({ ...prevScores, X: prevScores.X + 1 }));
+    } else if (winner === 'O') {
+      setScores(prevScores => ({ ...prevScores, O: prevScores.O + 1 }));
```

Comment:
"winner" blir en array genom setWinner([newWinnerSymbol, winningTiles]) på rad 97 i Board.jsx
Jag misstänker att du vill kolla efter:

```suggestion
    if (winner[0] === 'X') {
      setScores(prevScores => ({ ...prevScores, X: prevScores.X + 1 }));
    } else if (winner[0] === 'O') {
      setScores(prevScores => ({ ...prevScores, O: prevScores.O + 1 }));
```

Verkar funka fint annars!

Reply:
Yes. Stämmer. Har ändrats sedan innan.